Dizzy/Screens/Search Place/SearchPlaceCoordinator.swift

```diff
@@ -0,0 +1,59 @@
+//
```

Comment:
as i mentioned above, i don't think we need to have new coordinator for the search